radio/src/gui/480x272/zone.h

```diff
@@ -0,0 +1,54 @@
+
+#ifndef _ZONE_H_
+#define _ZONE_H_
+
+#include <inttypes.h>
+#include <string.h>
+
+#define LEN_ZONE_OPTION_STRING         8
+
+#define OPTION_VALUE_UNSIGNED(x)    setZoneOptionValue((uint32_t)x)
```

Comment:
Take a look at firmware.map. If you can find it there, then it means it is in a RAM (.data section to be precise.)